### PR TITLE
Added Lorenz Attractor WebVR example

### PR DESCRIPTION
**Pull request by paulmasson**
This is a math-based WebVR example. The Lorenz attractor is fairly easy to code, and while it's fun enough in a browser it's more interesting when you're right inside the evolution:

https://rawgit.com/paulmasson/three.js/webvr/examples/webvr_lorenz_attractor.html

The evolution will repeat and repeat, vanishing and restarting every time the color is red.

This is an example of the possibilities of WebVR for mathematical education and art.

### CODE REVIEW

File: examples/webvr_lorenz_attractor.html [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+			var camera, scene, renderer, x, y, z, c, geometry;
+
+			var scale = 30;
+			var multiplier = 5; // integer, increase for faster visualization
+
+			var steps = 100000;
+			var current = 1;
+			var shown = 10000;
+
+			var beta = 8/3;
```
Comment (cvan) [inline examples/webvr_lorenz_attractor.html]: nit: `8 / 3`

**Comment by cvan** [comment]
this is really beautiful. nice work! 🌃

**Comment by paulmasson** [comment]
@mrdoob no interest in this example?

**Comment by mrdoob** [comment]
Wouldn't it be better the lorenz attractor in front of the user?

**Comment by paulmasson** [comment]
@mrdoob not quite understanding: on my Oculus Go, it is right in front of me. Where is it for you, using what equipment?

**Comment by mrdoob** [comment]
Oh! Sorry... I didn't get to try it in VR. By checking the example on my desktop I had the impression the user was inside the object.

**Comment by mrdoob** [comment]
Okay, got to test it in VR.

It feels like the attractor is mapped into a sphere and we're inside of it...? The lorenz attractor doesn't seem to have depth... Is that intentional?

**Comment by paulmasson** [comment]
The viewer is close enough to the attractor to have to look around to see it all, but not inside it. I haven't done any particular mapping and there is no intention to flatten it. I personally perceive depth in the attractor. The black background makes it more dramatic, but maybe that flattens the object for you.

**Comment by mrdoob** [comment]
Trying to understand the code...

What is the size of the object? I can see the camera goes from 1 decimeter to 1 kilometer...?

**Comment by paulmasson** [comment]
The attractor is generally about 100 units across, but because it starts are random positions the outer loops can potentially be larger so I just threw in a bigger number. I tend to use 1000 by default, partly because it's in the documentation demo. Is there a significant performance hit if there's nothing there to render?

**Comment by mrdoob** [comment]
That's why I don't sense any depth. You're rendering an object 100 meters away that can be up to a 1 kilometer in size. Depth is not as obvious when going from 100 meters to 1 kilometer.

Do you mind making the object smaller? Place it 1 meter in front of the user and reduce its size until it's all visible. In VR it will become much more physical. Not only that but in positional VR you'll be able to walk around it 😊

**Comment by paulmasson** [comment]
Requested changes done. I didn't want to make the attractor too small so as not to lose too much detail. The perceived depth is much improved. Thanks!

**Comment by mrdoob** [comment]
Improved the example a little bit by adding a light and ground. https://github.com/mrdoob/three.js/commit/a75dbac0e6152e8a8adb5f13886d4f1aee7adeb6

**Comment by paulmasson** [comment]
I like it! Gives some context without distracting from the central object. Have to remember this technique.

Personally I prefer the attractor to be a bit larger. With the default resolution of current devices, it looks a bit too mushed together at this size. Then again, devices will improve.

**Pull request merged**